contracts/CO30_Munson_BreakAway.Enoch/initPlayerLocal.sqf

```diff
@@ -0,0 +1,17 @@
+#include "do_not_edit\script_component.hpp"
+#include "do_not_edit\initPlayerLocal.sqf"
+
+[
+    // Add additional files here
+    // if you create "edit_me/briefing/example.html, then add "Example" here
+] call mission_fnc_briefing;
+
+
+[intel_1, 2, false, 1, "Get Intel", // Edit this if you want to change the intel ACE action name
+[], 5,
+
+"STASH UPDATE", // Edit this for intel title
+
+"We have now moved 90% of the crates to .---- .---- ...-- , ----- ....- ..--- END. // Edit this for intel content
```

Comment:
```suggestion
"We have now moved 90% of the crates to .---- .---- ...-- , ----- ....- ..--- END." // Edit this for intel content
```

Reply:
This could be more interesting as an audio file playing from a radio / computer, I can help with that if you'd like

Converting morse code from text is trivial and will just take one of more people out of the game for a trivial conversion, having to transcribe audio will require more focus or co-operation